scripts/download/tools.mjs

```diff
@@ -229,6 +229,56 @@ export default async function main(platform) {
       expectedChecksum:  steveSHA,
       checksumAlgorithm: 'sha512'
     });
+
+  // Download Rancher Dashboard
+  const rancherDashboardVersion = 'v2.6.3-desktop.12';
+  const rancherDashboardURLBase = `https://github.com/rak-phillip/dashboard/releases/download/${ rancherDashboardVersion }`;
+  const rancherDashboardExecutable = 'rancher-dashboard-desktop-embed';
+  const rancherDashboardURL = `${ rancherDashboardURLBase }/${ rancherDashboardExecutable }.tar.gz`;
+  const rancherDashboardPath = path.join(resourcesDir, 'rancher-dashboard.tgz');
```

Comment:
Should this path be platform-specific? (`resourceDir` includes the platform, e.g. `win32`.)